app/assets/javascripts/tooltips.js

```diff
@@ -0,0 +1,3 @@
+/* globals $ */
+
+$(() => { $('[data-toggle="tooltip"]').tooltip(); });
```

Comment:
Wouldn't be enough with `$('[data-toggle="tooltip"]').tooltip();`? Removing the extra wrapping makes tests pass, but not sure what this code is exactly doing.